contracts/liquidityMining/KyberFairLaunchV3.sol

```diff
@@ -0,0 +1,609 @@
+// SPDX-License-Identifier: agpl-3.0
+pragma solidity 0.7.6;
+pragma abicoder v2;
+
+import {ReentrancyGuard} from '@openzeppelin/contracts/utils/ReentrancyGuard.sol';
+import {SafeMath} from '@openzeppelin/contracts/math/SafeMath.sol';
+import {SafeERC20} from '@openzeppelin/contracts/token/ERC20/SafeERC20.sol';
+import {IERC20Ext} from '@kyber.network/utils-sc/contracts/IERC20Ext.sol';
+import {PermissionAdmin} from '@kyber.network/utils-sc/contracts/PermissionAdmin.sol';
+import {IKyberFairLaunchV3} from '../interfaces/liquidityMining/IKyberFairLaunchV3.sol';
+import {GeneratedToken} from './GeneratedToken.sol';
+
+/// FairLaunch contract for Kyber DMM Liquidity Mining program
+/// Create a new token for each pool
+/// Allow stakers to stake LP tokens and receive reward tokens
+/// Allow extend or renew a pool to continue/restart the LM program
+/// When harvesting, rewards will be transferred to a RewardLocker
```

Comment:
Not updated comment, mentioning `RewardLocker`

Reply:
got it